File: packages/dd-trace/test/llmobs/parity/scenarios/openai-agents/simple-agent-streamed.py
import asyncio

from agents import Runner

from _agents_common import finish
from _agents_common import setup
from _agents_common import simple_agent


async def main():
    setup()
    result = Runner.run_streamed(simple_agent(), "What is the capital of France?")
    async for event in result.stream_events():
        del event
    finish()


asyncio.run(main())

File: packages/dd-trace/test/llmobs/parity/scenarios/openai-agents/simple-agent-streamed.js
'use strict'

const { setup, simpleAgent } = require('./common')

async function main () {
  const { agents, finish } = setup()
  const result = await agents.run(simpleAgent(), 'What is the capital of France?', { stream: true })
  for await (const event of result) void event
  await finish()
}

main().catch(error => {
  process.stderr.write(`${error.stack ?? error}\n`)
  process.exitCode = 1
})
